sql/explain.go

```diff
@@ -41,7 +41,7 @@ const (
 var explainStrings = []string{"", "debug", "plan", "trace", "types"}
 
 // Explain executes the explain statement, providing debugging and analysis
-// info about a DELETE, INSERT, SELECT or UPDATE statement.
+// info about an ALTER TABLE, CREATE, DELETE, DROP, INSERT, SELECT, or UPDATE statement.
```

Comment:
this list may change - is it really worth keeping? perhaps "info about the wrapped statement." is better.